### ISSUE TITLE
V2 bug: 'vars_prompt' has an invalid value ... and could not be converted to an dict.

### ISSUE THREAD

**Issue by dougluce** [ansible/ansible#10944]
```
% ansible-playbook test.yml
ERROR! the field 'vars_prompt' has an invalid value ([{u'prompt': u'Name this instance', u'name': u'name'}]), and could not be converted to an dict. Error was:
%
```

test.yml:

```
- hosts: web
  vars_prompt:
    - name: "important_var"
```

This is with ansible 2.0.0 (v2_final 198476e345) on Ubuntu 12.04.5 LTS

I checked back to 62d7956 -- same error there.  And yes, this does work on v1 current.

**Comment by kustodian** [comment]
I have the exact same problem on v2.

**Comment by jimi-c** [comment]
This was fixed in f3f3fb7c491effe9e61ae5a429ac796558c2963a. Thanks for reporting it, and let us know if you continue seeing any problems with it.
